### PR TITLE
Create sponsorship page

### PR DESCRIPTION
**Pull request by zachlatta**

### CODE REVIEW

**Comment by MaxWofford** [comment]
@zachlatta Could you check c9231ef and merge?

**Comment by MaxWofford** [comment]
Can this be added to production @zachlatta?

**Comment by MaxWofford** [comment]
I can merge this locally if it is ready.

File: app/assets/javascripts/pages.js.coffee [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+      center:
+        lat: 39
+        lng: -101
+      zoom: 3
+      scrollwheel: false
+    internal:
+      id: 'sponsor-map-jumbotron'
+    ->
+      handler.addMarkers markersJSON
+
```
Comment (zachlatta) [inline app/assets/javascripts/pages.js.coffee]: Can you make this addition to the bottom of the file? It's a clearer logical progression to have `pages#home` at the beginning.

File: app/assets/stylesheets/pages.css.scss [hunk 2]
```diff
@@ -13,6 +13,14 @@ $include-html-global-classes: false;
...
 }
 
+#sponsor-map-jumbotron {
+  height: 100px;
+
+  @media #{$large-up} {
+    height: 250px;
+  }
+}
+
```
Comment (zachlatta) [inline app/assets/stylesheets/pages.css.scss]: Can you give this a more descriptive name than `#sponsor-map-jumbotron`? This map isn't a jumbotron.

File: app/assets/stylesheets/pages.css.scss [hunk 3]
```diff
@@ -160,3 +168,26 @@ $include-html-global-classes: false;
   border-radius: 8px;
 }
 
+.pricing-table {
+  .invalid-item {
+    text-decoration: line-through;
+    color: #aeadad;
+  }
+  .price {
```
Comment (zachlatta) [inline app/assets/stylesheets/pages.css.scss]: Put new lines after each class here to follow the style in other files.

File: app/assets/stylesheets/pages.css.scss [hunk 4]
```diff
@@ -160,3 +168,26 @@ $include-html-global-classes: false;
...
+    background-color: #FAFAFA;
+  }
+  .title {
+    color: white;
+    font-size: 150%;
+    background-color: rgb(212, 89, 92);
+  }
+}
+.sponsorship {
```
Comment (zachlatta) [inline app/assets/stylesheets/pages.css.scss]: Again, new lines.

File: app/views/pages/sponsor.html.haml [hunk 5]
```diff
@@ -0,0 +1,107 @@
+= provide :title, 'Sponsor'
+
+.introduction.main-section
+  .row
+    %span.vertical-align
+      %h1
+        <b><span class="logo-h">h</span>ackEDU</b> empowers high school
```
Comment (zachlatta) [inline app/views/pages/sponsor.html.haml]: Perhaps make a partial for `<b><span class="logo-h">h</span>ackEDU</b>` if we're going to keep using this. Also, `<b>` is a style and therefore should be in CSS.

File: app/views/pages/sponsor.html.haml [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+        students <b>to build things they care about</b> through
+        <b>creating</b>, <b>connecting</b>, and growing <b>student-run</b>
+        coding clubs in <b>high schools</b>.
+
+.row
+  .small-11.small-centered.columns
+    %p
+      %span.h3 hackEDU
```
Comment (zachlatta) [inline app/views/pages/sponsor.html.haml]: It'd probably make more sense to put this inline with the rest of the body text. You can Google the best way to do this, but just writing `<span><h3>hackEDU</h3></span>` may not be too bad.

File: app/views/pages/sponsor.html.haml [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+      %li.bullet-item Logo on website/marketing materials
+      %li.bullet-item Blast four emails/Facebook messages to club leaders
+      %li.bullet-item Sixteen facilitated introductions to top hackers
+      %li.bullet-item Branded prize
+      %li.bullet-item Blurb on website
+      %li.bullet-item Ship swag to clubs
+      %li.bullet-item Opt-in email list
+
+%script
+  markersJSON = #{raw@clubs_hash.to_json}
```
Comment (zachlatta) [inline app/views/pages/sponsor.html.haml]: There should be a space between `raw` and `@clubs_hash.to_json`. See `pages/home.html.haml` for an example.

**Comment by zachlatta** [comment]
I made a couple comments, when you have a chance to review.

**Comment by zachlatta** [comment]
Can you add these ASAP?

**Pull request merged**